src/css/base.scss

```diff
@@ -1,7 +1,181 @@
 /* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: radial-gradient(circle, rgba(63,94,251,1) 0%, rgba(252,70,107,1) 100%);
-;
+
+/* Color Theme Swatches in HSLA */
+$Overlook-Hotel-favorite-1-hsla: hsla(15, 100, 30, 1);
+$Overlook-Hotel-favorite-2-hsla: hsla(12, 100, 58, 1);
+$Overlook-Hotel-favorite-3-hsla: hsla(37, 65, 82, 1);
+$Overlook-Hotel-favorite-4-hsla: hsla(175, 100, 30, 1);
+$Overlook-Hotel-favorite-5-hsla: hsla(175, 100, 46, 1);
+
+/* Color Theme Swatches in HSLA */
+$Overlook-Hotel-2-1-hsla: hsla(16, 96, 23, 1);
+$Overlook-Hotel-2-2-hsla: hsla(16, 91, 63, 1);
+$Overlook-Hotel-2-3-hsla: hsla(16, 96, 49, 1);
+$Overlook-Hotel-2-4-hsla: hsla(15, 50, 30, 1);
+$Overlook-Hotel-2-5-hsla: hsla(15, 95, 38, 1);
+
+$primary-background: radial-gradient(circle,  hsla(37, 65, 82, 1) 0%, hsla(15, 50, 30, 1) 100%);
```

Comment:
Come back to this at the end of the project if you have time! (create an issue for icebox!)

Reply:
Done!